docs/book/src/migration/v1vsv2.md

```diff
@@ -0,0 +1,61 @@
+# Kubebuilder v1 vs v2
+
+There are a number of changes in
+[controller-runtime](https://github.com/kubernetes-sigs/controller-runtime),
+[controller-tools](https://github.com/kubernetes-sigs/controller-tools)
+and [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder).
+
+## Common changes
+
+- They have migrated from `dep` to `go modules` for managing dependencies.
+
+## Controller-runtime
+
+- Automatic certificate generation for webhooks has been removed, and webhooks
```

Comment:
I think we can club webhook related changes together and move them below the `manager` and `client` changes because latter are more used more widely.

Reply:
Grouped webhook related stuff in a small section.